src/client/mod.rs

```diff
@@ -106,7 +95,7 @@ impl<Resolver, Policy: policy::Policy + Default> SrvClient<Resolver, Policy> {
 impl<Resolver: SrvResolver, Policy: policy::Policy> SrvClient<Resolver, Policy> {
     /// Gets a fresh set of SRV records from a client's DNS resolver, returning
     /// them along with the time they're valid until.
-    pub async fn get_srv_records(
+    async fn get_srv_records(
```

Comment:
?

Reply:
It doesn't filter the srv records, so users shouldn't call it.